client/my-sites/themes/index.web.js

```diff
@@ -26,20 +26,20 @@ export default function ( router ) {
 
 	const langParam = getLanguageRouteParam();
 	const routesWithoutSites = [
-		`/${ langParam }/themes/:tier(free|premium|marketplace)?`,
-		`/${ langParam }/themes/:tier(free|premium|marketplace)?/filter/:filter`,
-		`/${ langParam }/themes/:category(all|my-themes)?/:tier(free|premium|marketplace)?`,
-		`/${ langParam }/themes/:category(all|my-themes)?/:tier(free|premium|marketplace)?/filter/:filter`,
-		`/${ langParam }/themes/:vertical?/:tier(free|premium|marketplace)?`,
-		`/${ langParam }/themes/:vertical?/:tier(free|premium|marketplace)?/filter/:filter`,
+		`/${ langParam }/themes/:tier(free|premium|marketplace)?/:collection?`,
+		`/${ langParam }/themes/:tier(free|premium|marketplace)?/filter/:filter/:collection?`,
+		`/${ langParam }/themes/:category(all|my-themes)?/:tier(free|premium|marketplace)?/:collection?`,
+		`/${ langParam }/themes/:category(all|my-themes)?/:tier(free|premium|marketplace)?/filter/:filter/:collection?`,
+		`/${ langParam }/themes/:vertical?/:tier(free|premium|marketplace)?/:collection?`,
+		`/${ langParam }/themes/:vertical?/:tier(free|premium|marketplace)?/filter/:filter/:collection?`,
 	];
 	const routesWithSites = [
-		`/${ langParam }/themes/:tier(free|premium|marketplace)?/:site_id(${ siteId })`,
-		`/${ langParam }/themes/:tier(free|premium|marketplace)?/filter/:filter/:site_id(${ siteId })`,
-		`/${ langParam }/themes/:category(all|my-themes)?/:tier(free|premium|marketplace)?/:site_id(${ siteId })`,
-		`/${ langParam }/themes/:category(all|my-themes)?/:tier(free|premium|marketplace)?/filter/:filter/:site_id(${ siteId })`,
-		`/${ langParam }/themes/:vertical?/:tier(free|premium|marketplace)?/:site_id(${ siteId })`,
-		`/${ langParam }/themes/:vertical?/:tier(free|premium|marketplace)?/filter/:filter/:site_id(${ siteId })`,
+		`/${ langParam }/themes/:tier(free|premium|marketplace)?/:collection?/:site_id(${ siteId })`,
+		`/${ langParam }/themes/:tier(free|premium|marketplace)?/filter/:filter/:collection?/:site_id(${ siteId })`,
+		`/${ langParam }/themes/:category(all|my-themes)?/:tier(free|premium|marketplace)?/:collection?/:site_id(${ siteId })`,
+		`/${ langParam }/themes/:category(all|my-themes)?/:tier(free|premium|marketplace)?/filter/:filter/:collection?/:site_id(${ siteId })`,
+		`/${ langParam }/themes/:vertical?/:tier(free|premium|marketplace)?/:collection?/:site_id(${ siteId })`,
+		`/${ langParam }/themes/:vertical?/:tier(free|premium|marketplace)?/filter/:filter/:collection?/:site_id(${ siteId })`,
```

Comment:
I think the problem (or one of the problems) here is that we optionally match a `:collection` parameter, while we want to match the actual `collection` word. 🤔 

It's not an easy fix, I think, because of how complex the routing of the showcase is.

GPT suggests `(/collection)?` instead, but I haven't tried.

Reply:
GPT's suggestion didn't seem to work 😄. I'm still looking into this.

Reply:
I found the solution 🥳.

Fixed